### ISSUE TITLE
Semantic annotations

### ISSUE THREAD

**Issue by Glamdring** [inukshuk/jekyll-scholar#172]
Do you think it would be good if you also semantically annotate publications (for example with JSON-LD like this: http://pastebin.com/84enS1iq ) - it would make it easier for search engines, including specialized ones like Google Scholar to locate open scientific resources better.

I've recently written an overview on the problems of scientific publishing (https://techblog.bozho.net/broken-scientific-publishing-model-attempt-improve/) and I think the semantic aspect is one way to improve the situation.

**Comment by inukshuk** [comment]
Absolutely, that's a great idea. The best approach, probably, would be to add a json-ld export to bibtex-ruby: we already have citeproc, bibtexml, and rdf export there so that would be the most natural thing to do. Exposing this via jekyll-scholar should be easy and a big improvement.

**Comment by sneakers-the-rat** [comment]
Aha, glad to have found this issue. I'm interested in adding semantic annotations to citations as well, any thoughts on using RDFa vs JSON-LD? I was thinking RDFa could embed into the existing tags generated to render the citation, but haven't taken a look yet at what's already implemented in bibtex-ruby.
